lookup-table.yaml

```diff
@@ -212,3 +212,15 @@
     - utility
     - library
     - api
+- id: lambdabettergrass
+  names:
+    - lambdabettergrass
+  tags:
+    - decoration
+    - cosmetic
+    - adventure-rpg
+    - vanilla+
```

Comment:
I'm not sure if vanilla+ is allowed (@NebelNidas ?). Try using `vanilla-plus` instead.